src/utils/ConfigLoader.ts

```diff
@@ -51,6 +52,15 @@ export const getExpressAppConfig = (directory: string) => (app: express.Applicat
                 visuallyHiddenText: accessibleName
             }
         }
+    };
+
+    const url = getEnv('PIWIK_URL');
+    const site = getEnv('PIWIK_SITE_ID');
```

Comment:
is the PIWIK_SITE_ID the same for all apps?

Reply:
Nope, every env has different site ID. For example we use 26 in Rebel and 10 in Staging.

Reply:
ok, i've already seen this has been configured in the global env properties 👍